pkgs/applications/misc/xmenu/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchgit, libX11, libXft, imlib2 }:
+
+{
```

Comment:
Does this work? I think you’ve missed `stdenv.mkDerivation`.

Reply:
Surprisingly it does work for me, but I will change it.

Reply:
ah, what happened was I developed this in an overlay and then copied it into nixpkgs and made this formatting mistake. It "worked" because my overlay was still active... oops.